src/placeholders/placeholder.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+var util = require('./../util');
+
+/**
+ * Placeholder
+ *
+ * Representation of the placeholder to be placed over blank block elems
+ *
+ */
+
+function Placeholder(elem, tracker) {
+  var tag = elem.tagName;
+  var text = tracker.placeholderTexts[tag] || "Placeholder";
+
+  var me = this;
+  me.elem = elem;
+  me.container = tracker.container;
+
+  me.holder = document.createElement("div");
+  me.holder.className = "qed-placeholder";
+  me.holder.innerHTML = "<" + tag + ">" + text + "</" + tag + ">";
```

Comment:
Any reason why string building instead of dom building?

Also, please rename "text" to "html" to make it clear that it's html, or, if it is meant to be just text, then use createTextNode()

Looking at the doc for addPlaceholder it seems like you probably want the latter.  Maybe later, we can support the former by allowing a different type for the cotnent argument to addPlaceholder, e.g. if string, then plain text, if dom element, then clone it, etc.